src/earn/EarnActivePool.tsx

```diff
@@ -33,26 +36,33 @@ function PoolDetailsLoading() {
   )
 }
 
-export default function EarnActivePool() {
+interface Props {
+  buttonDisplay: 'ExitAndDeposit' | 'ViewPools'
```

Comment:
optional naming suggestion
```suggestion
  cta: 'ExitAndDeposit' | 'ViewPools'
```

Reply:
Fixed in 8048d97b30997219688e8560a26ab8f2a65fe2c2!